python/ray/dag/compiled_dag_node.py

```diff
@@ -1887,6 +1887,129 @@ async def execute_async(
         self._execution_index += 1
         return fut
 
+    def visualize(self, filename="compiled_graph", format="png", view=False):
+        """
+        Visualize the compiled graph using Graphviz.
+
+        This method generates a graphical representation of the compiled graph,
+        showing tasks and their dependencies.
+
+        **When to Call:**
```

Comment:
I think you can just mention this needs to be called after `experimental_compile()`, and then do actual checking in code

Reply:
fixed!